src/coreModules/api/specEndpointFactory/index.js

```diff
@@ -0,0 +1,34 @@
+import openApi from 'dina-schema/build/openApi.json'
+
+const buildOperationIdPathnameMap = openApiSpec => {
+  const map = {}
+  Object.keys(openApiSpec.paths).forEach(pathname => {
+    const verbs = openApiSpec.paths[pathname]
+    Object.keys(verbs).forEach(verbName => {
+      const verb = verbs[verbName]
+      if (verb.operationId)
+        map[verb.operationId] = {
+          pathname,
+          verbName,
+        }
+    })
+  })
+  return map
+}
+
+const map = buildOperationIdPathnameMap(openApi)
+
+export const buildEndpointSpec = ({ operationId, ...rest }) => {
+  if (!map[operationId]) {
+    throw new Error(`Operation id: ${operationId} unknown`)
+  }
+  const { pathname } = map[operationId]
+
+  return {
+    key: operationId,
+    pathname,
+    ...rest,
+  }
+}
+
+export { openApi }
```

Comment:
might be more clear what is exported if you remove `export` from `const buildEndpointSpec` and add it here instead